src/clr-addons/quick-list/quick-list.component.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { CLR_BLANK_OPTION, ClrQuickListValue } from './add-option/add-option.service';
+
+@Component({
+  selector: 'clr-quick-list',
+  templateUrl: './quick-list.component.html',
+  styleUrls: ['./quick-list.component.scss'],
+})
+export class ClrQuickListComponent<T> {
+  @Input() blankOption: ClrQuickListValue<T> = CLR_BLANK_OPTION;
+  @Input() allValues: Array<ClrQuickListValue<T>> = [this.blankOption];
+  @Input() values: Array<ClrQuickListValue<T>> = [this.blankOption];
+  @Input() addLabel: string = 'ADD';
+  @Input() mandatory: boolean = false;
+  @Output() valuesChanged = new EventEmitter<Array<ClrQuickListValue<T>>>();
+  @Output() addClicked = new EventEmitter<void>();
```

Comment:
this event emitter is not needed, as valuesChanged already emits every change on the values

Reply:
could be beneficial in case of automatic scoll down when clicking add